cmd/table.go

```diff
@@ -164,21 +180,44 @@ func writeTable(tableData [][]string, params tableCommandParams) error {
 	table.AppendBulk(tableData)
 	table.Render()
 
-	if !params.writeToReadme {
-		_, err := os.Stdout.Write(buf.Bytes())
-
-		return err
-	}
+	return buf
+}
 
-	file, err := os.ReadFile("README.md")
-	if err != nil {
-		return err
-	}
+func renderREADME(buf bytes.Buffer) string {
+	file := string(must(os.ReadFile("README.md")))
 
-	first := strings.Split(string(file), "<!-- RULES_TABLE_START -->")[0]
-	last := strings.Split(string(file), "<!-- RULES_TABLE_END -->")[1]
+	first := strings.Split(file, "<!-- RULES_TABLE_START -->")[0]
```

Comment:
💭 Looks like a good occasion for using https://pkg.go.dev/strings#CutPrefix and https://pkg.go.dev/strings#CutSuffix? But really, it's fine and working as-is, no need to change stuff.

Reply:
I don't think that would work here, as those strings don't "start with prefix" or suffix but are rather somewhere deep inside of the README?

Reply:
Hmm I might be wrong 😅 Yeah, looks like I misread the docs. We could do two cuts, but it. doesn't get much better -- https://play.golang.com/p/u8OtalFaste